Project.toml

```diff
@@ -6,6 +6,7 @@ version = "0.7.33"
 ChainRulesCore = "d360d2e6-b24c-11e9-a2a3-2a2ae2dbcce4"
 ChainRulesTestUtils = "cdddcdb0-9152-4a09-a978-84456f9df70a"
 Compat = "34da2185-b29b-5c13-b0c7-acf172513d20"
+FiniteDifferences = "26cc04aa-876d-5657-8c51-4c34ba976000"
```

Comment:
That doesn't look right. I don't think we want to add FiniteDifferences as a dependency?

Reply:
Good catch